tests_integration/__tests__/__snapshots__/early-exit.js.snap

```diff
@@ -84,7 +84,7 @@ Format options:
 
   --no-bracket-spacing     Do not print spaces between brackets.
   --jsx-bracket-same-line  Put > on the last line instead of at a new line.
-  --parser <flow|babylon|typescript|postcss|json|graphql>
+  --parser <flow|babylon|typescript|postcss|postcss-less|postcss-scss|json|graphql>
```

Comment:
Suggestion: let's just name it `less` and `scss`. And allow both `postcss` and `css` to work and only mention `css` in the docs.

We're unlikely going to ever support anything else than postcss to parse our CSS and at the end of the day the users don't care, they just want to parse their css/less/sass.

Reply:
Do you mean that it is unlikely that we'll support two different parsers for the same language, like we do for JS with both Babylon and Flow? Or that it is unlikely that we'll every switch from postcss to something else?

I have been researching switching away from postcss:

- [css-tree](https://github.com/csstree/csstree) for CSS
- [less](https://github.com/less/less.js) or [gonzales-pe](https://github.com/tonyganch/gonzales-pe/) for Less
- [dart-sass](https://github.com/sass/dart-sass/) or [gonzales-pe](https://github.com/tonyganch/gonzales-pe/) for SCSS (see also https://github.com/sass/dart-sass/issues/88)

Reply:
If we did switch away, we probably wouldn't want to keep supporting postcss (another advantage of using --language)

Reply:
Yeah, what I mean is that we just name it with the language, so we don't get stuck to a specific parser if we want to switch.